src/properties.c

```diff
@@ -277,6 +277,39 @@ static int get_lower_limit(int fd, char *group_id, int value_type,
 	return rc;
 }
 
+static int set_limit(int fd, const char *group_id, const char *key,
+		     int value_type, knot_value_type value)
+{
+	int rc;
+
+	switch (value_type) {
+	case KNOT_VALUE_TYPE_INT:
+		rc = storage_write_key_int(fd, group_id, key, value.val_i);
+		break;
+	case KNOT_VALUE_TYPE_FLOAT:
+		rc = storage_write_key_float(fd, group_id, key, value.val_f);
+		break;
+	case KNOT_VALUE_TYPE_BOOL:
+		rc = storage_write_key_bool(fd, group_id, key, value.val_b);
```

Comment:
I left it because as the KNoT protocol does not support it, virtualthing will not receive any messages of this type. However, with this implementation, virtualthing is ready to receive messages with Boolean limits (storage is fully implemented).